ntc_templates/templates/cisco_asa_show_running-config_access-group.textfsm

```diff
@@ -0,0 +1,8 @@
+Value ACCESS_LIST (\S+)
+Value INTERFACE (\S+)
+
+Start
+  ^access-group\s+${ACCESS_LIST}\s+in\s+interface\s+${INTERFACE}\s* -> Record
+  ^. -> Error "Did not match any entries"
+
+EOF
```

Comment:
* Since we're matching the entire line anyways, we can also add `$$` to anchor to the end of line too.
   * (Tested still works with the test data. Good job!)
* EOF is inherent so we don't need to explicitly call it out. (Saves us a few bits and bytes.)

```suggestion
  ^access-group\s+${ACCESS_LIST}\s+in\s+interface\s+${INTERFACE}\s*$$ -> Record
  ^. -> Error "Did not match any entries"
```